### PR TITLE
Update Seedlet readme.

### PR DESCRIPTION
**Pull request by kjellr**
This PR simplifies the README. It's a starting point for #2172, stripping out the outdated parts and narrowing down to just the minimum for now.

### CODE REVIEW

File: seedlet/README.md [hunk 1]
```diff
@@ -1,19 +1,29 @@
...
+Seedlet is simple, text-driven WordPress theme.
 
-## What does it do?
+![](https://github.com/Automattic/themes/raw/master/seedlet/screenshot.png)
 
-When you reduce a theme design down to a set of systematic design decisions, you end up with something called a _Style Guide_. The Seedlet system works by taking the rules of a Style Guide and expressing them through carefully placed variables or _design tokens_ that influence the appearance of a WordPress site. This unifies the design decisions needed to style the aesthetic appearance of Gutenberg Blocks, the theme Header + Footer areas, WooCommerce, Jetpack and more. It also syncs styles between the editor and the frontend so that you don’t need to hand-write CSS for both. This greatly speeds up the Gutenberg theme development process and reduces the amount of manual styling that typically goes into developing a theme. 
+Along with being an excellent standalone theme, Seedlet is designed to be a parent-theme that allows folks to quickly create Gutenberg-ready child-themes for WordPress. *[Instructions for how to make child themes are coming soon]*
 
-### What controls does the system come with?
+## Technical Overview
```
Comment (jffng) [inline seedlet/README.md:9]:
```suggestion
## Development

To make changes to the styles, [node.js](https://nodejs.org/en/) is recommended:

1. `npm i` to install the required development dependencies
2. `npm start` to start the development server
3. Make changes in `assets/sass` 

## Technical Overview
```

File: seedlet/README.md [hunk 2]
```diff
@@ -1,19 +1,29 @@
...
 
 - **Fonts** - Font-family, size, weight, and line-height rules. 
 - **Colors** - Primary, secondary, background, foreground and border colors. 
 - **Spacing** - A default 8px vertical rhythm between all blocks and major components. It also includes utility spacing classes for negative margins.
 - **Responsive Logic** - Built-in responsive behavior across Blocks and Components.
 
-## How does it work?
+### Structure
 
 The system itself lives in the `/seedlet/sass` directory as a collection of Sass partials broken up by scope and hierarchy. The partials get compiled down to singular CSS files that live in the main Seedlet directory and cascade downward like so:
```
Comment (jffng) [inline seedlet/README.md:28]:
```suggestion
The system itself lives in the `/assets/sass` directory as a collection of Sass partials broken up by scope and hierarchy. The partials get compiled down to singular CSS files that live in the main Seedlet directory and cascade downward like so:
```

Comment (jffng) [reply seedlet/README.md:28]: Same goes for the **Frontend** and **Editor** director references:

`/seedlet/variables.css` should be `/assets/variables.css`
`/seedlet/style.css` --> `style.css`

**Comment by kjellr** [comment]
Thanks, @jffng! Would you mind taking one more look?

**Pull request merged**